src/System.IO.Packaging/tests/Tests.cs

```diff
@@ -20,6 +20,8 @@ public class Tests
         private const string Mime_MediaTypeNames_Image_Jpeg = "image/jpeg"; // System.Net.Mime.MediaTypeNames.Image.Jpeg
         private const string s_DocumentXml = @"<Hello>Test</Hello>";
         private const string s_ResourceXml = @"<Resource>Test</Resource>";
+        private static readonly bool s_IsWindowsSystem = Path.DirectorySeparatorChar == '\\';
+        private static readonly string s_LocalFile = s_IsWindowsSystem ? @"c:/resources/image1.jpg" : "/resources/image1.jpg";
```

Comment:
Nit: s_LocalFile => s_localFile (or maybe s_fullPathToLocalFile, or something like that)